spec/index.bs

```diff
@@ -1355,16 +1355,16 @@ dictionary GPUBufferDescriptor : GPUObjectDescriptorBase {
 <script type=idl>
 typedef [EnforceRange] unsigned long GPUBufferUsageFlags;
 interface GPUBufferUsage {
-    const GPUBufferUsageFlags MAP_READ      = 0x0001;
-    const GPUBufferUsageFlags MAP_WRITE     = 0x0002;
-    const GPUBufferUsageFlags COPY_SRC      = 0x0004;
-    const GPUBufferUsageFlags COPY_DST      = 0x0008;
-    const GPUBufferUsageFlags INDEX         = 0x0010;
-    const GPUBufferUsageFlags VERTEX        = 0x0020;
-    const GPUBufferUsageFlags UNIFORM       = 0x0040;
-    const GPUBufferUsageFlags STORAGE       = 0x0080;
-    const GPUBufferUsageFlags INDIRECT      = 0x0100;
-    const GPUBufferUsageFlags QUERY_RESOLVE = 0x0200;
+    const unsigned long MAP_READ      = 0x0001;
```

Comment:
that's super sad: we are making a typedef just so we do *not* use it :(
should we at least make `typedef unsigned long GPUFlags` to use there?

Reply:
The typedef mainly exists to be used in faraway input slots like GPUBufferDescriptor. Your proposed typedef would be fine with me though.

Reply:
Done